api/db-migrations/000012_add_traffic_rule_name.down.sql

```diff
@@ -0,0 +1,30 @@
+UPDATE router_versions
```

Comment:
Can we update the down query to follow the same workflow as the up query? Currently, this is also missing the `traffic_rule ORDER BY position` as I added in the up query (but forgot to mention) which will ensure the rule ordering is preserved.